### PR TITLE
login

### PR DESCRIPTION
**Pull request by halakhamis07**

### CODE REVIEW

**Comment by Zahimues** [comment]
you forgot to fetch by email when the log is false
if u can do it tell me if u dont want i will

**Comment by Karyum** [comment]
> you forgot to fetch by email when the log is false if u can do it tell me if u dont want i will

@Zahimues that should be a different pull request, we will do it later

File: src/pages/Login/index.jsx [hunk 1]
```diff
@@ -1,30 +1,61 @@
 import style from './style.module.css'
 // React Wavify
 import Wave from 'react-wavify'
+import { useState, useRef } from 'react'
```
Comment (Karyum) [inline src/pages/Login/index.jsx]: useRef is no being used

File: src/pages/Login/index.jsx [hunk 2]
```diff
@@ -1,30 +1,61 @@
...
+import axios from 'axios'
 
 function Login(props) {
+  const [client, setClient] = useState({
+    gov_id: '',
```
Comment (Karyum) [inline src/pages/Login/index.jsx:13]: change `gov_id` to `govId`

I know it's annoying but there are other changes on this pull request so I said why not

File: src/pages/Login/index.jsx [hunk 3]
```diff
@@ -1,30 +1,61 @@
...
+  })
+  const [loading, setLoading] = useState(false)
+  const [error, setError] = useState(false)
+
+  function fetchLogin() {
+    setLoading(true)
+    axios.post('http://localhost:4000/api/client/login', client).then((res) => {
+      setLoading(false)
+      if (!res.data) {
+        setError('error with fetch')
```
Comment (Karyum) [inline src/pages/Login/index.jsx]: `Please try again, or login by phone/email`

File: src/pages/Login/index.jsx [hunk 4]
```diff
@@ -1,30 +1,61 @@
...
+      } else {
+        window.localStorage.setItem('access_token', res.data.access_token)
```
Comment (Karyum) [inline src/pages/Login/index.jsx:23]: use `react-router-dom` to move the user to the homepage once the request is done

**Review by Karyum — CHANGES_REQUESTED** [review]

**Review by Karyum — APPROVED** [review]

**Pull request merged**